src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -703,11 +703,12 @@ weekly_daily_flag_plots("archery", "Temperature", "slope flag suspect")
 Histograms of all parameter/site combos
 ```{r}
 # generate basic histograms for dfs
-generate_histograms <- function(df, df_index) {
+generate_general_histogram <- function(df, df_index) {
   
   site_param <- toupper(sub("-", " ", df_index, fixed = TRUE))
   title <- paste("Histogram of ", site_param)
```

Comment:
can we add a (n= number of measurements)?

Reply:
I don't see why not, I'll add that to the histogram issue.